EngineLayer/FdrAnalysis/PsmData.cs

```diff
@@ -1,14 +1,62 @@
 ﻿using Microsoft.ML.Data;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
 
 namespace EngineLayer.FdrAnalysis
 {
     public class PsmData
     {
+        public static readonly IImmutableDictionary<string, string[]> trainingInfos = new Dictionary<string, string[]>
+        {
+            { "standard", new [] { "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", "Notch", "PsmCount", "ModsCount", "MissedCleavagesCount", "Ambiguity", "LongestFragmentIonSeries", "HydrophobicityZScore", "IsVariantPeptide" } },
+            { "topDown", new [] { "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", "Notch", "PsmCount", "ModsCount", "Ambiguity", "LongestFragmentIonSeries" } }
+        }.ToImmutableDictionary();
+
+        /// <summary>
+        /// These are used for percolator. Trainer must be told the assumed direction for each attribute as it relates to being a true positive
+        /// Here, a weight of 1 indicates that the probability of being true is for higher numbers in the set.
+        /// A weight of -1 indicates that the probability of being true is for the lower numbers in the set.
+        /// </summary>
+        public static readonly IImmutableDictionary<string, int> assumedAttributeDirection = new Dictionary<string, int> {
```

Comment:
I'm a little surprised delta mass isn't on here.

Reply:
me too. deltamass was not an effective means of separating true from false